spinalcordtoolbox/reports/slice.py

```diff
@@ -38,26 +38,34 @@ class Slice(object):
 
     def __init__(self, images, p_resample=0.6):
         """
-        :param images: list of 3D volumes to be separated into slices.
+        :param images: list of 3D or 4D volumes to be separated into slices.
         """
         logger.info('Resample images to {}x{} mm'.format(p_resample, p_resample))
-        self._images = list()
+        self._images = list()  # 3d volumes
+        self._4d_images = list()  # 4d volumes
+        self._image_seg = None  # for cropping
+        self._absolute_paths = list()  # Used because change_orientation removes the field absolute_path
```

Comment:
Adding a link here to https://github.com/neuropoly/spinalcordtoolbox/issues/3304 to remind us to revisit.